Renewed the Installation article (#213)
AdguardTeam/KnowledgeBase: docs/adguard-for-android/installation.md

## System  requirements

**OS version:** Android 7.0 or higher

**RAM:** at least 2 GB

**Free disk space:** 500 MB

## Installation

Most Android-based apps are distributed via Google Play; however, AdGuard is not presented there, because Google prohibits distribution of network-level ad blockers via Google Play, i.e. apps that block commercials in other apps. You will find more information about Google restrictive policy [in our blog](https://blog.adguard.com/en/google-removes-adguard-android-app-google-play/).

That’s why you can only install AdGuard for Android manually. To use the app on your mobile device, you will need to do the following.

1. **Download the app on your device**. Here are a few ways you could go about this:

>     * head over to [our website](https://adguard.com/en/adguard-android/overview.html) and tap the *Download* button

Author: доне

    * start the browser and type in the following URL: [https://adguard.com/apk](https://adguard.com/apk)
    * or scan this QR code

    ![QR code *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/android/installation/inst_qr.png)

2. **Allow installing apps from unknown sources**. Once the file download is complete, tap on the *Open* option in the notification. 

![Installing apps from unknown sources *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/android/installation/inst_1.png)

> A popup will then appear, prompting you to adjust *Settings*. Tap on *Settings*, navigate to *Install unknown apps*, and grant permission for the browser you've used to download the file.

Reviewer: ```suggestion
A popup will appear. Tap *Settings*, navigate to *Install unknown apps*, and grant permission for the browser you've used to download the file.
```

![Installing apps from unknown sources *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/android/installation/inst_3.png)

This tutorial is right for Android 8+. If you have an older OS version, before downloading the apk file, open *Settings*, then select *Additional settings* in *System & Device*. Switch on *Unknown sources* and tap *OK* in the system warning window.

> 3. **Install the app**. Once the browser secures the necessary permission, the system will ask you if you want to install the AdGuard app. Tap *Install*.

Reviewer: ```suggestion
3. **Install the app**. Once the browser has obtained the necessary permissions, the system will ask you if you want to install the AdGuard application. Tap *Install*.
```

![Installing apps from unknown sources *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/android/installation/inst_4.png)

> You will then be asked to read the *License agreement* and AdGuard's *Privacy Policy*. You can also participate in product development. To do this, you should check boxes *Send crash reports automatically* and *Send technical and interaction data*. Tap *Continue* after that.

Reviewer: ```suggestion
You will then be asked to read AdGuard's *License Agreement* and *Privacy Policy*. You can also participate in product development. To do this, check the boxes for *Send crash reports automatically* and *Send technical and interaction data*. Then tap *Continue*.
```

![Privacy Policy *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/android/installation/fl_3.png)

> 4. **Create a local VPN**. To filter all the traffic right on your device and not to rout it through a remote server AdGuard needs to set up a VPN connection. 

Reviewer: ```suggestion
4. **Create a local VPN**. In order to filter all traffic directly on your device and not route it through a remote server, AdGuard needs to establish a VPN connection.
```

![Create a local VPN *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/android/installation/fl_2.png)

5. **Enable HTTPS filtering**. It is not a mandatory option, however, we advice to turn it on for the best ad-blocking quality.

> If your device runs on Android 7-9, after setting up a local VPN, you'll be prompted with a screen to install a root certificate and configure HTTPS filtering. 

Reviewer: ```suggestion
If your device is running Android 7–9, you'll be prompted to install a root certificate and configure HTTPS filtering after the local VPN setup.
```

![Enable HTTPS filtering on Android 7-9 *mobile_border](https://cdn.adtidy.org/content/kb/ad_blocker/android/installation/cert_1.jpg)

> After you hit *Install now*, a prompt will pop up requesting you to authenticate the certificate installation using a password or fingerprint.

Reviewer: ```suggestion
After you tap *Install now*, a prompt will appear asking you to authenticate the certificate installation with a password or fingerprint.
```
